### ISSUE TITLE
Grass is blinking and game crashes upon teleporting (post bog witch update)

### ISSUE THREAD

**Issue by VilVik12** [zolantris/ValheimMods#142]
### What happened?

As said in the title, with the last updated version of valheim raft and latest valheim update, grass is blinking and cmd is flooded by error messages (MissingMethodException) when near a raft , the game then crash after teleporting with a portal on a raft.

### Steps to reproduce

1.Enter a world
2.Be near a raft from valheimRaft
3.Watch Bepinex LogOutPut

### Log outputs

```txt
[Error  : Unity Log] MissingMethodException: Method not found: void .WearNTear.Destroy(HitData)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.AddPiece (ZNetView netView, System.Boolean isNew) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
ValheimVehicles.Vehicles.VehiclePiecesController.ActivatePiece (ZNetView netView) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
ValheimVehicles.Vehicles.VehiclePiecesController.InitPiece (ZNetView netView) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
ValheimRAFT.Patches.ZNetView_Patch.ZNetView_Awake (ZNetView __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) ZNetView.DMD<ZNetView::Awake>(ZNetView)
UnityEngine.Object:Instantiate(GameObject, Vector3, Quaternion)
ZNetScene:CreateObject(ZDO)
ZNetScene:CreateObjectsSorted(List`1, Int32, Int32&)
ZNetScene:CreateObjects(List`1, List`1)
ZNetScene:DMD<ZNetScene::CreateDestroyObjects>(ZNetScene)
ZNetScene:Update()

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Info   : Unity Log] 10/30/2024 17:00:44: Force generating hmap (2112.00, 0.00, 1152.00)

[Info   : Unity Log] 10/30/2024 17:00:44: Force generating hmap (2176.00, 0.00, 1152.00)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Info   : Unity Log] 10/30/2024 17:00:44: Spawning dungeon

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ValheimVehicles.Patches.VehicleMovementPatches.Update (MonoUpdaters __instance) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomLateUpdate (System.Single deltaTime) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomLateUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::LateUpdate>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
ValheimVehicles.Vehicles.VehiclePiecesController.CustomUpdate (System.Single deltaTime, System.Single time) (at <d290e8f9a13e4e2183dda06a02c96bd8>:0)
MonoUpdatersExtra.CustomUpdate (System.Collections.Generic.List`1[T] container, System.Collections.Generic.List`1[T] source, System.String profileScope, System.Single deltaTime, System.Single time) (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) MonoUpdaters.DMD<MonoUpdaters::Update>(MonoUpdaters)

[Error  : Unity Log] MissingMethodException: Method not found: Vector2i .ZoneSystem.GetZone(UnityEngine.Vector3)
Stack trace:
```

### Environment

```markdown
OS:
- **OS**: Windows 10
- **Graphics**: DX11 (or Vulkan)
Mods:
- **BepInEx**: 5.4.2202
- Jotunn v2.21.2
- **ValheimRAFT**: ValheimRAFT-v2.3.0-beta.4
```

**Comment by EthanWS6** [comment]
I am having this same issue. All grass "blinks" aggressively even when not around a placed boat piece.  For me, any boat piece I placed could not be deleted either.

**Comment by zolantris** [comment]
Fixed >= 2.4.1
